kernel/packages/unity-interface/BrowserInterface.ts

```diff
@@ -343,6 +344,14 @@ export class BrowserInterface {
   }
 
   async RequestGIFProcessor(data: { imageSource: string; id: string; isWebGL1: boolean }) {
+    // tslint:disable-next-line
+    const isSupported = typeof OffscreenCanvas !== "undefined"
```

Comment:
Have you tested this in Firefox? Keep in mind that `OffscreenCanvas` seems to be undefined only in worker context, and not in the main thread.

Also, we should test if the OffscreenCanvas has `2d context` support.

Reply:
I'll add the `typeof OffscreenCanvasRenderingContext2D === "function"` check, thx for pointing it out!